main.tf

```diff
@@ -145,12 +145,7 @@ data "aws_ami" "ecs_ami" {
 
 data "aws_ami" "user_ami" {
   count       = "${var.lookup_latest_ami ? 0 : 1}"
-  most_recent = true
```

Comment:
What is the significance of removing this?

Reply:
`most_recent` only has value if the AMI lookup returns a list with more than one entry. Since we filter by `image-id`, there will be at most one entry, so I don't think `most_recent` is necessary here.